models/uniswap/uniswap_schema.yml

```diff
@@ -15,6 +15,11 @@ models:
       - &blockchain
         name: blockchain
         description: "Blockchain which the DEX is deployed"
+      - &dex_project_name
```

Comment:
i don't think you intended to add these two columns to `uniswap_trades` which feeds into `dex.trades`?

Reply:
yeah in the wrong table, resolving